### PR TITLE
Feature/readme docs

### PR DESCRIPTION
**Pull request by ConradoClark**
#### What problem is this solving?

Adding full documentation to README.md

#### How to test it?

Nothing to test, just type check if possible :)

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,9 +1,320 @@
...
+**dockId**: Id of the VTEX dock.
+**dockName**: Name of the VTEX dock.
+**entityType**: Entity Type (Company, Individual, Government, etc)
+**icmsTaxPayer**: Defines if the entity is an ICMS taxpayer.
+**pCredSN**: Rate for calculating the use of ICMS credit.
+**neighborhood**: Neighborhood of the establishment's address.
+**state**: State of the establishment's address.
+**stateTaxId**: State registration of the company, if it is an ICMS tax payer.
+**street**: Street address of the establishment.
+**streetNumber**: Address number of the establishment
```
Comment (pscruzzz) [inline docs/README.md:91]: Just adding a point at the end 😅
```suggestion
**streetNumber**: Address number of the establishment.
```

File: docs/README.md [hunk 2]
```diff
@@ -1,9 +1,320 @@
...
+**suframa**: Suframa number ("Superintendencia da Zona Franca de Manaus").
+**taxRegime**: Company's tax regime (simplified, real profit, estimated profit, and so on)
```
Comment (pscruzzz) [inline docs/README.md:93]: Just adding a point at the end 😅
```suggestion
**taxRegime**: Company's tax regime (simplified, real profit, estimated profit, and so on).
```

**Comment by pscruzzz** [comment]
Really well written doc!!!!

File: docs/README.md [hunk 3]
```diff
@@ -1,9 +1,320 @@
...
+```
+
+Since these are private entities, it is also necessary to have authorization to add this information to MasterData, so authorization parameters must be placed in the header:
+
+**VtexIdclientAutCookie**: To obtain this value you must open the Developer Tools on your browser, on the Applications tab, while inside the VTEX Admin. Then look for Cookie values and copy the value of the VtexIdclientAutCookie cookie value.
+
+**AppKey/AppToken**: To get the AppKey and AppToken, log into the admin and go to the "Account Management" tab and then click on "Account". You can see all registered tokens in the security area. You can create one by clicking on "Generate appKey and appToken", then fill in the request information and click on "Generate new tokens".
+
+#### Using Avatax BR
+After finishing the setupand activating the connector your store will be able to calculate and display taxes and fees through Avalara.
```
Comment (gabrielHosino) [inline docs/README.md:293]: I think it needs to separate words here "setup and"

**Review by gabrielHosino — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 3 above)
Comment (ConradoClark) [reply docs/README.md]: Fixed in https://github.com/vtex-apps/avatax-br-admin/pull/21/commits/c62af4ed0f97e41f7e924ef9716e410fb66f0a26

**Review by gabrielHosino — APPROVED** [review]

**Pull request merged**